Resources/Textures/Structures/Wallmounts/barsign.rsi/meta.json

```diff
@@ -7,6 +7,7 @@
   "license": "CC-BY-SA-3.0",
   "copyright": "https://github.com/vgstation-coders/vgstation13/commit/1fbb5e417d0200fe60c6b628656a371093d2d715",
   "copyright": "https://github.com/tgstation/tgstation/commit/68aade356cf53032f206cb5a887581ac7d166cae",
+  "copyright": "whiskeyechoes by CerebralError#4852",
```

Comment:
There's no way this is valid JSON. How does this even pass the linter?

Despite that, I think I prefer having this over dealing with a merge conflict in the future.